static/index.html

```diff
@@ -124,6 +124,9 @@
           <li class="nav-item">
             <a class="nav-link" href="faq.html">FAQ</a>
           </li>
+          <li class="nav-item">
```

Comment:
I just noticed this nav-item for Forum is only present on the index.html website. I think you will have to manually add it to every html file we have.

Reply:
Done. Good catch!